.github/workflows/python-package.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
```

Comment:
I assume that you'd like to continue testing 3.6 and 3.7 for as long as they remain supported?

Reply:
Thanks for catching that; I added those back in 8b2a42a